proposal-templates/0000-swiftpm-template.md

```diff
@@ -0,0 +1,64 @@
+# Package Manager Feature name
+
+* Proposal: [SE-NNNN](NNNN-filename.md)
+* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+
+*During the review process, add the following fields as needed:*
+
+* Implementation: [apple/swift-package-manager#NNNNN](https://github.com/apple/swift-package-manager/pull/NNNNN)
+* Decision Notes: [Rationale](https://forums.swift.org/), [Additional Commentary](https://forums.swift.org/)
+* Bugs: [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN), [SR-MMMM](https://bugs.swift.org/browse/SR-MMMM)
+* Previous Revision: [1](https://github.com/apple/swift-evolution/blob/...commit-ID.../proposals/NNNN-filename.md)
+* Previous Proposal: [SE-XXXX](XXXX-filename.md)
+
+## Introduction
+
+A short description of what the feature is. Try to keep it to a single-paragraph
+"elevator pitch" so the reader understands what problem this proposal is
+addressing.
+
+Swift-evolution thread: [Discussion thread topic for that
+proposal](https://forums.swift.org/)
+
+## Motivation
+
+Describe the problems that this proposal seeks to address. If the problem is
+that some common pattern is currently hard to express, show how one can
+currently get a similar effect and describe its drawbacks. If it's completely
+new functionality that cannot be emulated, motivate why this new functionality
+would help Swift developers create better Swift packages.
+
+## Proposed solution
+
+Describe your solution to the problem. Provide examples and describe how they
+work. Show how your solution is better than current workarounds: is it cleaner,
+easier, or more efficient?
+
+## Detailed design
+
+Describe the design of the solution in detail. If it involves adding or
+modifying a functionality in the package manager, explain how the package
+manager behave in different scenarios and with existing features. If it's a new
+API in the `Package.swift` manifest, show the full API and its documentation
+comments detailing what it does.  The detail in this section should be
+sufficient for someone who is *not* one of the authors to be able to reasonably
+implement the feature.
+
+## Security
+
+Will this change impact security of the build process and other package manager
```

Comment:
What would you think of tweaking this to: "Does this change have any impact on security, safety, or privacy?"